enterprise/kernel/src/test/java/org/neo4j/kernel/enterprise/builtinprocs/ListQueriesProcedureTest.java

```diff
@@ -71,36 +77,11 @@ public void shouldContainTheQueryItself() throws Exception
     public void shouldProvideElapsedCpuTime() throws Exception
     {
         // given
-        CountDownLatch nodeLocked = new CountDownLatch( 1 ), listQueriesLatch = new CountDownLatch( 1 );
-        Node node;
-        try ( Transaction tx = db.beginTx() )
-        {
-            node = db.createNode();
-            node.setProperty( "v", 1L );
-            tx.success();
-        }
-        threads.execute( parameter ->
-        {
-            try ( Transaction tx = db.beginTx() )
-            {
-                tx.acquireWriteLock( node );
-                nodeLocked.countDown();
-                listQueriesLatch.await();
-            }
-            return null;
-        }, null );
-        nodeLocked.await();
-
-        threads.executeAndAwait( parameter ->
-        {
-            db.execute( "MATCH (n) SET n.v = n.v + 1" ).close();
-            return null;
-        }, null, waitingWhileIn( GraphDatabaseFacade.class, "execute" ), 5, SECONDS );
-
-        try
+        String QUERY = "MATCH (n) SET n.v = n.v + 1";
```

Comment:
I used uppercase as a highlight of the central aspect of the test, but I don't mind changing it.